### ISSUE TITLE
How to convert a node of xml to json array

### ISSUE THREAD

**Issue by relax-admin** [stleary/JSON-java#595]
This is achieved in python, is it possible to add similar parameters in java

In the following example, **interfaces are parsed into an array** as expected

``` python
import json

import xmltodict

t1 = '''
<servers>
  <server>
    <name>host1</name>
    <os>Linux</os>
    <interfaces>
      <interface>
        <name>em0</name>
        <ip_address>10.0.0.1</ip_address>
      </interface>
    </interfaces>
  </server>
</servers>
'''
content = xmltodict.parse(t1, force_list=['servers', 'interfaces'])
data_str = json.dumps(content)
print(data_str)

```
output
``` json
{
  "servers": [
    {
      "server": {
        "name": "host1",
        "os": "Linux",
        "interfaces": [
          {
            "interface": {
              "name": "em0",
              "ip_address": "10.0.0.1"
            }
          }
        ]
      }
    }
  ]
}
```
is it possible to add similar parameters in java?
``` java
public JSONObject parse(String content, List<String> forceList) {
        // to be
        return  org.json.XML.toJSONObject(content);
    }
```

**Comment by stleary** [comment]
Hi @relax-admin Sorry for the late reply. XML-to-JSON transformations are imperfect. We use `XmlParserConfiguration` to fine-tune the results. Feel free to post a pull request to implement the desired behavior.

**Comment by stleary** [comment]
I think this is fixed in #646
